content/tutorials/react-native-presence-publish-subscribe.textile

```diff
@@ -0,0 +1,539 @@
+---
+title: Build a realtime chat app with react native and ably
```

Comment:
This should be `React Native` and `Ably`

Reply:
Fixed this, thanks